modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/index/sorted/io/SortedIndexTreeIo.java

```diff
@@ -48,17 +55,47 @@
  * </ul>
  */
 public interface SortedIndexTreeIo {
-    /** Offset of the index column link (6 bytes). */
-    int INDEX_COLUMNS_LINK_OFFSET = 0;
+    /** Item size without index columns in bytes. */
+    int ITEM_SIZE_WITHOUT_COLUMNS = Short.SIZE // Inlined index columns size.
+            + PARTITIONLESS_LINK_SIZE_BYTES // Index columns link.
+            + 2 * Long.BYTES; // Row ID.
 
-    /** Offset of rowId's most significant bits, 8 bytes. */
-    int ROW_ID_MSB_OFFSET = INDEX_COLUMNS_LINK_OFFSET + PARTITIONLESS_LINK_SIZE_BYTES;
+    /** Index columns are not fully inlined, the index column size is {@link #indexColumnsInlineSize()}. */
```

Comment:
I don't understand, how is is constant expected to be used? Maybe it should be called `NOT_FULLY_INLINED_FLAG`? And why is it a `short`?

Reply:
Maximum inline size (**InlineUtils#MAX_BINARY_TUPLE_INLINE_SIZE**) is 2 KB, i.e. will not be more than `short`.
What exactly is not clear?

Reply:
`the index column size is {@link #indexColumnsInlineSize()}`
I believe that this statement is incorrect. Or it's not clear what is meant by `index column size`

Reply:
`the index columns size is {@link #indexColumnsInlineSize()}` ?

Reply:
Fix it